API:ENH: Adding raw traces as an output for the functional interface

swkeemink opened the pull request:
It would be useful for new users to quickly get started with implementing FISSA, and an important part of that is comparing the raw extracted traces to the decontaminated ones. Previously the functional interface did not allow for that, and with this PR I make that a standard behavior. Now with deltaf calculation: 
```
deltaf_result, deltaf_raw = fissa.run_fissa(images_location, rois_location, return_deltaf=True)
```

and without
```
result, raw = fissa.run_fissa(images_location, rois_location, return_deltaf=False)
```

See also https://github.com/rochefort-lab/fissa/pull/169 for more discussion on this.

For now this is a draft pull request, because #228 should be finished first so that I can implement those changes in the notebook easily, and then update the plotting accordingly.

scottclowe commented on fissa/core.py:
```diff
@@ -1260,8 +1260,19 @@ def run_fissa(
+        If ``return_deltaf=True``, this is Δf/f\ :sub:`0`; otherwise it's
+        the raw extracted signal.
+
+        .. versionadded:: 1.0.0
+
```
You don't need to add versionadded to unreleased functions/methods. The version where `raw` is added is the same as the version where `run_fissa` is added.

```suggestion
```

swkeemink replied:
Argh, of course. Updated that.

swkeemink commented:
I have now updated the notebook to compare both the raw and separated traces. I also added an example to show how to inspect the neuropil region signals.
